src/Features/SupportLegacyModels/Tests/PublicPropertyHydrationAndDehydrationUnitTest.php

```diff
@@ -876,6 +876,20 @@ public function test_it_ignores_the_key_if_the_model_does_not_exist()
         Livewire::test(ModelsComponent::class, ['model' => $model])
             ->call('$refresh');
     }
+
+    public function test_it_returns_null_if_the_model_was_deleted()
+    {
+        $this->expectNotToPerformAssertions();
+
+        $model = Author::create(['id' => 1, 'title' => 'foo', 'name' => 'bar', 'email' => 'baz']);
+
+        $component = Livewire::test(ModelsComponent::class, ['model' => $model])
+            ->call('$refresh');
```

Comment:
Please change this to instead have an `->assertSuccessful()` after the call and remove the `$this->expectNotToPerformAssertions();` 

```suggestion
            ->call('$refresh')
            ->assertSuccessful();
```

Reply:
There may be use-cases when someone will prefer to handle the missing model different, like showing a custom message or re-creating the model.

Reply:
Done!